src/__tests__/integration/basic/arweave-gateway-interaction-loader.test.ts

```diff
@@ -0,0 +1,71 @@
+import { writeFileSync } from 'fs';
```

Comment:
This will be removed form integrations tests, just to show you @ppedziwiatr  example that it works